DeviceManager/LoggerHandler.py

```diff
@@ -0,0 +1,58 @@
+from flask import Blueprint, request, jsonify, make_response
+
+from DeviceManager.conf import CONFIG
+from DeviceManager.Logger import Log
+
+from DeviceManager.app import app
+
+from DeviceManager.SerializationModels import parse_payload, load_attrs
+from DeviceManager.SerializationModels import log_schema
+from DeviceManager.utils import format_response, HTTPRequestError, get_pagination
+
+logger = Blueprint('log', __name__)
+LOG = Log()
+
+class LoggerHandler:
+    
+    def _init_(self):
+        pass
+    
+    @staticmethod
+    def update_log_level(req):
+        _, json_payload = parse_payload(req, log_schema)
+        LOG.update_log_level(json_payload['level'].upper())
+
+        result = {
+            'result': 'ok'
+        }
+
+        return result
+    
+    @staticmethod
+    def get_log_level():
+        result = {
+            'level': LOG.get_log_level()
+        }
+
+        return result
+
+
+@logger.route('/log', methods=['PUT'])
+def flask_update_log_level():
+    try:
+        result = LoggerHandler.update_log_level(request)
```

Comment:
If the result is always a success, why do I need a result?

Reply:
I reviewed and doesn't really make sense with the purpose. Initially, I thinked that the result would be manipulated. The new PR, removes this variable and just return if the request was a success or not.